doc/reference/modules/toolset_guide.xml

```diff
@@ -262,23 +262,23 @@ new SchemaExport(cfg).Create(false, true);]]></programlisting>
                     </thead>
                     <tbody>
                     <row>
-                        <entry><literal>hibernate.connection.driver_class</literal></entry>
+                        <entry><literal>connection.driver_class</literal></entry>
                         <entry>jdbc driver class</entry>
                     </row>
                     <row>
-                        <entry><literal>hibernate.connection.url</literal></entry>
+                        <entry><literal>connection.url</literal></entry>
                         <entry>jdbc url</entry>
```

Comment:
This part of the documentation is commented out, so I have just removed the prefix without fixing other Hibernate specific discrepancies.